src/app/CommandHandlerInterface.h

```diff
@@ -232,5 +234,79 @@ class CommandHandlerInterface
     CommandHandlerInterface * mNext = nullptr;
 };
 
+class CommandHandlerInterfaceShim : public CommandHandlerInterface
+{
+public:
+    using CommandHandlerInterface::CommandHandlerInterface;
+    typedef Loop (*CommandIdCallback)(CommandId id, void * context);
+    typedef Loop (*CommandEntryCallback)(DataModel::AcceptedCommandEntry id, void * context);
```

Comment:
This is not what the old interface is though: old interface only returned an `id` and this is the error we are trying to fix.

If we provide a compatibility shim, we have to provide one that converts the id to an entry ... and that is the hard/awkard part.